lib/rules/lint-no-shadowed-elements.js

```diff
@@ -21,8 +21,9 @@ module.exports = class NoShadowedElements extends Rule {
 
         let firstChar = node.tag.charAt(0);
         let startsWithUpperCase = firstChar === firstChar.toUpperCase() && firstChar !== firstChar.toLowerCase();
+        let containsDot = node.tag.includes('.');
 
-        if (!startsWithUpperCase) {
+        if (!startsWithUpperCase && !containsDot) {
```

Comment:
This should be caught by the `this.isLocal(node)` check above 🤔

Reply:
Hmm, actually I think I mis-read things a bit. This fix seems good to me, sorry about that...